utils/network_test.go

```diff
@@ -90,8 +92,37 @@ func TestNetworkSet(t *testing.T) {
 		n := new(utils.Network)
 		require.ErrorIs(t, n.Set("blah"), utils.ErrUnknownNetwork)
 	})
+
+	t.Run("custom network - success", func(t *testing.T) {
```

Comment:
Note: this is failing to unmarshal the networkJSON string into a network type (even when only contains name field)